src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.search-bar {
+  position: relative;
+  align-items: center;
+  margin-top: 20px;
+}
+
+.search-bar__icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  display: flex;
+  margin-top: 20px;
+  margin-bottom: 20px ;
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-bar__input:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  text-shadow: 1px 4px 4px rgba(0, 0, 0, 0.25);
+  font-weight: 900;
+  outline: none;
+}
+
+.search-bar__icon--small {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-bar__input--small {
+  height: 42px;
+  width: 100%;
+  padding-left: 33px;
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
```

Comment:
you don't need to write font-family in all elements, add only once in html selector